.github/workflows/nodejs.yml

```diff
@@ -19,10 +21,10 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
-        node-version: [14, 16]
+        node-version: [14, 16, 18]
 
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v1
```

Comment:
If you do want to use relative versions, this would need to get updated to v3 as well.